ipa_room_exploration/ros/src/room_exploration_action_client.cpp

```diff
@@ -63,79 +84,30 @@ int main(int argc, char **argv)
 	}
 	std::cout << "map-size: " << map.rows << "x" << map.cols << std::endl;
 
-//	const std::string topic = "/move_base/global_costmap/costmap";
-//	nav_msgs::OccupancyGrid grid;
-//	grid = *(ros::topic::waitForMessage<nav_msgs::OccupancyGrid>(topic, nh));
-//	ROS_INFO("got grid");
-//
-//	std::vector<signed char> dats;
-//	dats = grid.data;
-//
-//	std::cout << dats.size() << std::endl;
-//	int s = 200;
-//	cv::Mat test_map = cv::Mat(s, s, map.type());
-//
-//	for(size_t u = 0; u < test_map.cols; ++u)
-//	{
-//		for(size_t v = 0; v < test_map.rows; ++v)
-//		{
-//			test_map.at<uchar>(u,v) = (uchar) dats[v+u*s];
-//		}
-//	}
-//
-//	cv::imshow("testtt", test_map);
-//	cv::waitKey();
-
 	ROS_INFO("Waiting for action server to start.");
 	// wait for the action server to start
 	ac.waitForServer(); //will wait for infinite time
 
 	ROS_INFO("Action server started, sending goal.");
 
-	DynamicReconfigureClient drc_exp(nh, "room_exploration_server/set_parameters", "room_exploration_server/parameter_updates");
-	drc_exp.setConfig("room_exploration_algorithm", 2);
-//	drc_exp.setConfig("path_eps", 3);
```

Comment:
please keep the commented drc_exp entries, they are there for documentary reasons for new users.